templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Are you sure you want to delete the car {{ object.model }}?</h1>
+  <form method="post">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Yes, delete</button>
+  </form>
+  <a href="{% url 'taxi:car-list' %}" class="btn btn-primary">No, cancel</a>
```

Comment:
The cancel link currently redirects to the car list. It should redirect to the manufacturer list instead, using `{% url 'taxi:manufacturer-list' %}`.